### PR TITLE
Ask user to overwrite if output file exists

### LINKED ISSUE #114: Force writing to recording.mp4 with no arguments
Hi I lost a recording of a couple of hours because I started wf-recorder without params. 
I think this is a bug. We should default to not overwrite and ask the user if interactive. 
See
https://wiki.archlinux.org/index.php?title=Screen_capture&diff=next&oldid=634963

### PR DESCRIPTION
**Pull request by soreau**
Fixes #114.

### CODE REVIEW

**Review by ammen99 — COMMENTED** [review]
If the user specifies `-f` do we still want to ask ?

File: src/main.cpp [hunk 1]
```diff
@@ -376,6 +377,24 @@ void handle_sigint(int)
...
 }
 
+static bool user_specified_overwrite(std::string filename)
+{
+    struct stat buffer;   
+    if (stat (filename.c_str(), &buffer) == 0)
+    {
+        std::string input;
+        std::cout << "Output file \"" << filename << "\" exists. Overwrite? Y/n: ";
+        std::getline(std::cin, input);
```
[the remark this answers is not in the document]
Comment (soreau) [reply src/main.cpp:387]: You are right, 'Y' does indicate it is the default and in fact it is. When pressing enter alone, it 'succeeds' and starts recording.

**Comment by soreau** [comment]
> If the user specifies `-f` do we still want to ask ?

Yes.

File: src/main.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ammen99) [reply src/main.cpp:387]: Oh indeed, I missed the first condition.

**Comment by ammen99** [comment]
Thanks!

**Pull request merged**